model.py

```diff
@@ -3923,6 +3923,11 @@ def to_search_documents(cls, works):
 
         _db = Session.object_session(works[0])
 
+        # If this is a batch of search documents, postgres needs extra working
+        # memory to process the query quickly.
+        if len(works) > 50:
+            _db.execute("set work_mem='200MB'")
```

Comment:
This is the command I got from Ashok. It only applies to the current database session.